Marlin/src/pins/pins_FYSETC_F6_13.h

```diff
@@ -29,7 +29,8 @@
 #endif
 
 #ifdef SD_DETECT_INVERTED
-  #error "SD_DETECT_INVERTED must be disabled for the FYSETC_F6_13 board."
+  #undef  SD_DETECT_INVERTED
+  #warning "SD_DETECT_INVERTED must be disabled for the FYSETC_F6_13 board."
```

Comment:
Few users ever see warnings. It is better to force them to be aware of their settings.